src/flask_container_scaffold/util.py

```diff
@@ -68,15 +67,10 @@ def parse_input(logger, obj, default_return=BaseApiView):
     :returns: Instantiated object of type obj on success, or default_return
               on failure to parse.
     """
+    args_dict = _preprocess_request()
     try:
-        if request.is_json:
-            parsed_args = obj.model_validate_json(json.dumps(request.json))
-        else:
-            if request.args:
-                args = request.args
-            else:
-                args = request.form
-            parsed_args = obj.model_validate_json(json.dumps(args.to_dict()))
+        # Unpack the dict into keyword arguments
+        parsed_args = obj(**args_dict)
     except ValidationError as e:
         logger.error(f"Validation error is: {e}")
         errors_result = {}
```

Comment:
e.errors_count() Is added at pydantic 2, What about changing that to len(e.errors()) ? So this works for pydantic v 1.10.